sql/src/test/java/io/crate/integrationtests/AggregateExpressionIntegrationTest.java

```diff
@@ -95,8 +95,8 @@ public void test_filter_in_aggregate_expr_with_group_by_single_numeric_column_wi
                 "   [1, 1, null, null]) AS t(x, y) " +
                 "GROUP BY y");
         assertThat(printedTable(response.rows()),
-                   is("[2]\n" +
```

Comment:
shall we order by maybe to make it deterministic?

Reply:
The column is an array and we can't order by arrays.

The sort is "stable" in that we output nulls now first without explicit order by, so the test won't be flaky (before we always returned it last)